src/Aspire.Hosting/MongoDB/MongoDbBuilderExtensions.cs

```diff
@@ -0,0 +1,92 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Sockets;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Publishing;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding MongoDB resources to an <see cref="IDistributedApplicationBuilder"/>.
+/// </summary>
+public static class MongoDbBuilderExtensions
+{
+    private const int DefaultContainerPort = 27017;
+    private const string PasswordEnvVarName = "MONGO_INITDB_ROOT_PASSWORD";
+
+    /// <summary>
+    /// Adds a MongoDB container to the application model. The default image is "mongo" and the tag is "latest".
+    /// </summary>
+    /// <returns></returns>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <param name="port">The host port for MongoDB.</param>
+    /// <param name="password">The password for the MongoDB root user. Defaults to a 'password' password.</param>
```

Comment:
Looks like it actually defaults to `null`, unless MongoDB itself does that when no password is specified in the connection string?